glue-codes/fast-farm/LESinflow/FAST.Farm.fstf

```diff
@@ -63,6 +64,14 @@ DEFAULT            Mod_WakeDiam       - Wake diameter calculation model (-) (swi
 DEFAULT            C_WakeDiam         - Calibrated parameter for wake diameter calculation (-) [>0.0 and <0.99] or DEFAULT [DEFAULT=0.95] [unused for Mod_WakeDiam=1]
 DEFAULT            Mod_Meander        - Spatial filter model for wake meandering (-) (switch) {1: uniform, 2: truncated jinc, 3: windowed jinc} or DEFAULT [DEFAULT=3]
 DEFAULT            C_Meander          - Calibrated parameter for wake meandering (-) [>=1.0] or DEFAULT [DEFAULT=1.9]
+--- CURLED-WAKE PARAMETERS [only used if Mod_Wake=2 or 3] ---
+True          Swirl                   - Switch to include swirl velocities in wake [only used if Mod_Wake=2 or Mod_Wake=3] (-) (switch) [DEFAULT=TRUE]
+0             k_VortexDecay           - Vortex decay constant for curl (-) [DEFAULT=0]
+DEFAULT       NumVortices             - The number of vortices in the curled wake model (-) [DEFAULT=100]
+DEFAULT       sigma_D                 - The width of the vortices in the curled wake model non-dimesionalized by rotor diameter (-) [DEFAULT=0.2]
+DEFAULT       FilterInit              - Switch to filter the initial wake plane deficit and select the number of grid points for the filter {0: no filter, 1: filter of size 1} or DEFAULT [DEFAULT=1] [unused for Mod_Wake=1] (switch)

```

Comment:
No need to say, "[unused for Mod_Wake=1]". This section is only used if Mod_Wake=2 or Mod_Wake=3.